cmake/nokia_ngage.cmake

```diff
@@ -0,0 +1,133 @@
+cmake_minimum_required(VERSION 3.16)
+
+enable_language(CXX)
+
+set(GCC_MODULE_COMN_DEFS -D__SYMBIAN32__ -D__GCC32__ -D__EPOC32__ -D__MARM__ -D__MARM_ARMI__)
+set(GCC_MODULE_MODE_DEFS -DNDEBUG -D_UNICODE)
```

Comment:
Is `_UNICODE` required for the n-gage sdk? If so, also move it to the n-gage sdk CMake toolchain file.

`-DNDEBUG` is configuration (debug/release) dependent, so must go completely. CMake adds this automatically.

Reply:
I will remove `NDEBUG`.

`_UNICODE` is required internally by the S60 C++ API. Currently moving as much as I can to the toolchain file.